packages/interpretations/src/components/Lists/CommentsList.js

```diff
@@ -154,9 +155,17 @@ export class CommentsList extends React.Component {
 
         return (
             <div className={this.props.classes.commentSection}>
-                {ViewMoreReplies}
-                {Comments}
-                {InputField}
+            {ViewMoreReplies}
+            {Comments}
+            {InputField}
+            {this.state.deleteDialogIsOpen && (
+                <DeleteDialog
+                    title={i18n.t('Delete comment')}
+                    text={i18n.t('Are you sure you want to delete this comment?')}
+                    onDelete={() => this.onDeleteComment(this.state.comment)}
```

Comment:
`this.state.comment` is being passed to `onDeleteComment `  but it looks like`onDeleteComment` doesn't take any input params? (row 63: `onDeleteComment() {`)

Reply:
Right, another thing left over. Cleaning up.